Recommend to check `symbolicator` if source maps are not working on Self-Hosted (#11765)
getsentry/sentry-docs: platform-includes/sourcemaps/legacy-uploading-methods/javascript.mdx

Sometimes build scripts and plugins produce pre-compressed minified files (for example, webpack's [compression plugin](https://github.com/webpack/compression-webpack-plugin)). In these cases, you'll need to disable such plugins and perform the compression **after** the generated source maps/source files have been uploaded to Sentry.

### (Self-Hosted Sentry) Verify the `symbolicator` service is operating normally

> 
> If you are running a self-hosted version of Sentry, verify that the `symbolicator` service/container is operating normally.
> You can do so by checking the container's logs.

Reviewer: ```suggestion

If you're running a self-hosted version of Sentry, you can verify that the `symbolicator` service/container is operating normally by checking the container's logs.
```
